CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # bedrock-vue-wallet ChangeLog
 
+## 21.2.1 - 2023-10-xx
+
+### Added
+- Add "Settings" and "Log Out" options to header & drawer.
```

Comment:
This looks good and is an improvement on wide screens. What happens on narrow screens?

Reply:
(Just added this to the PR content above as well)

Items get moved over to the app drawer with the other navigation links

<img src="https://github.com/digitalbazaar/bedrock-vue-wallet/assets/56396286/c49f71b9-f387-4202-b991-75abfe7ec938" width="275" />

<img src="https://github.com/digitalbazaar/bedrock-vue-wallet/assets/56396286/6456c1d9-4317-43cd-b670-fdffb3dff7b8" width="275" />